ietf/templates/doc/document_html.html

```diff
@@ -31,6 +31,15 @@
             <script src="{% static 'ietf/js/document_html.js' %}"></script>
             <script src="{% static 'ietf/js/theme.js' %}"></script>
         {% endif %}
+            <style>
+                {# Try and recolor black colors in SVGs for dark mode. This will need further tweaking. #}
+                svg *[fill='black'], svg text:not([fill]) {
+                    fill: currentColor;
+                }
+                svg *[stroke='black'] {
+                    stroke: currentColor;
+                }
```

Comment:
You will need to redo white as well.  Some SVGs explicitly include white boxes, as I discovered...

See https://github.com/martinthomson/rfc-css/blob/d6c36ce15c5cc2663dd7500509dd7edc36d6e8d1/rfc.css#L824-L836